lib/page-object/sections.rb

```diff
@@ -0,0 +1,29 @@
+module PageObject
+  class SectionCollection
```

Comment:
This class just provides niceties for searching for sections that meet certain criteria.